### DISCUSSION TITLE
remote based theia not opening in electron webview

### DISCUSSION THREAD

**Discussion by AbhinandanPun** [eclipse-theia/theia#9983] [Q&A]
Hi,

I am building a cloud orchestrator, theia is set up in aws. On logging in to the custom made login form, the API provides socket addresses like:-
3.178.2.1:2000 for some user
3.178.2.1:2001 for some user 

each running an instance of the theia container.

the socket address is very well accessible in chrome browser, moz, or edge browser. SS below with inspect.
![image](https://user-images.githubusercontent.com/23399844/131229592-c808816d-211a-4f8c-883d-1cf55400279a.png)

but it is not opening inside the electron webview or even in the browserwindow of electron
![image](https://user-images.githubusercontent.com/23399844/131229620-88d5f729-48ab-4794-8d84-7268e6fe7c96.png)

please help me work this issue out ASAP.

**Comment by msujew** [comment]
Hi @Abhinandan14, displaying the browser version of Theia inside of electron is not something we support explicitely, see #9751 for more discussion on that topic.

But maybe we can still help you find a solution for that issue. Does your browser log any errors trying to access the embedded Theia app? Any `TypeError` instances or something similiar?

**Reply by AbhinandanPun** [reply]
hey how are you,

No i there is no error
can you help me please

**Reply by vince-fugnitto** [reply]
@Abhinandan14 I don't think it's possible, the webview (embedded) would not have access to things like local-storage, browser apis and so on. In addition, it is quite hard to help without a way to easily reproduce the setup.

**Comment by AbhinandanPun** [comment]
Hello @msujew and thank you for replying.

And no there are no errors logging in neither in chrome nor in electron browser.
